gamemodes/cinema/gamemode/modules/theater/sh_video.lua

```diff
@@ -206,15 +206,18 @@ if SERVER then
 end
 
 if CLIENT then
+	local TrustConvar = CreateClientConVar("cinema_load_all_urls","0",true,false,"",0,1)
+    local TEMP_TRUST = {}
```

Comment:
I think we should remove the TRUSTED_VIDEO_HOST function below, and just change it to a table, which contains the urls like dropbox by default, and when F8 is pressed, the current host gets added to that same table. Then update the services to look in the TRUSTED_VIDEO_HOST table rather than calling the TRUSTED_VIDEO_HOST function.